src/app/components/BCP47Tag.svelte

```diff
@@ -12,7 +12,7 @@
   $: displayEnterBCP47Tag = bcp47Tag == "" || validTag;
   $: {
     if (validTag) {
-      worker.updateBCP47Tag(bcp47Tag);
+      worker.setProjectData({ languages: [{ name: "", id: bcp47Tag }] });
```

Comment:
We may use the bcp47Tag to generate the language name, or in an opposite way (let the user to input language name and we generate the bcp47Tag).